src/Goerli-DssSpell.t.sol

```diff
@@ -283,23 +307,23 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertEq(castTime, spell.eta());
     }
 
-    function test_OSMs() public {
-        address OAZO_ADDR = 0x0F1AE882272032D494926D5D983E4FBE253CB544;
+    // function test_OSMs() public {
+    //     address OAZO_ADDR = 0x0F1AE882272032D494926D5D983E4FBE253CB544;
 
-        assertEq(OsmAbstract(addr.addr("PIP_ETH")).bud(OAZO_ADDR), 0);
-        assertEq(OsmAbstract(addr.addr("PIP_WBTC")).bud(OAZO_ADDR), 0);
-        assertEq(OsmAbstract(addr.addr("PIP_WSTETH")).bud(OAZO_ADDR), 0);
+    //     assertEq(OsmAbstract(addr.addr("PIP_ETH")).bud(OAZO_ADDR), 0);
+    //     assertEq(OsmAbstract(addr.addr("PIP_WBTC")).bud(OAZO_ADDR), 0);
+    //     assertEq(OsmAbstract(addr.addr("PIP_WSTETH")).bud(OAZO_ADDR), 0);
 
-        vote(address(spell));
-        spell.schedule();
-        hevm.warp(spell.nextCastTime());
-        spell.cast();
-        assertTrue(spell.done());
+    //     vote(address(spell));
+    //     spell.schedule();
+    //     hevm.warp(spell.nextCastTime());
+    //     spell.cast();
+    //     assertTrue(spell.done());
 
-        assertEq(OsmAbstract(addr.addr("PIP_ETH")).bud(OAZO_ADDR), 1);
-        assertEq(OsmAbstract(addr.addr("PIP_WBTC")).bud(OAZO_ADDR), 1);
-        assertEq(OsmAbstract(addr.addr("PIP_WSTETH")).bud(OAZO_ADDR), 1);
-    }
+    //     assertEq(OsmAbstract(addr.addr("PIP_ETH")).bud(OAZO_ADDR), 1);
+    //     assertEq(OsmAbstract(addr.addr("PIP_WBTC")).bud(OAZO_ADDR), 1);
+    //     assertEq(OsmAbstract(addr.addr("PIP_WSTETH")).bud(OAZO_ADDR), 1);
+    // }
```

Comment:
Might be better to use the `private` trick instead (which was very clever)

Reply:
I've cleanup previous specific tests and used the `private` trick to match with spells-mainnet in 22103ea.
Also, I've disabled auth tests in specific tests as was done in the previous spell on mainnet to save time on testing, as there are no new chainlog entries (including sources) to check wards against deployers, in afb86b6.

Reply:
I've added test templates for OSMs and Medianizers specific tests in 67bcab3